sentry-rails/lib/generators/sentry_generator.rb

```diff
@@ -0,0 +1,18 @@
+require "rails/generators/base"
+
+class SentryGenerator < ::Rails::Generators::Base
+  class_option :dsn, type: :string, desc: "Sentry DSN"
+
+  def copy_initializer_file
+    dsn = options[:dsn] ? "'#{options[:dsn]}'" : "ENV['SENTRY_DSN']"
+
+    create_file "config/initializers/sentry.rb", <<~RUBY
+      # frozen_string_literal: true
+
+      Sentry.init do |config|
+        config.breadcrumbs_logger = [:active_support_logger]
+        config.dsn = #{dsn}
```

Comment:
we'd like to enable performance by default, so let's add
```ruby
config.enable_tracing = true
```

Reply:
I was concerned that enabling tracing by default without setting any sampling rate/logic would cause unexpected quota consumption. But it's hard to set a good sampling strategy statically, so I decided to not go for it first.

Reply:
I've updated it though.

Reply:
We're generally trying to move away from client side sampling and let people control their sampling dynamically on the server. Of course there are nuances for very large apps with 100% sampling so we will not get rid of the client side sampling configs, but we do want to encourage them less for newer users.